ReleaseHistory.md

```diff
@@ -12,9 +12,11 @@
 - FND => False negative reduction in dynamic phase
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
+- DEP => upgrade dependency versions
 - NEW => new feature 
 
 ## UNRELEASED
+* DEP: Eliminate build warning - The .NET SDK has newer analyzers with version '7.0.1' than what version of '7.0.0' of 'Microsoft.CodeAnalysis.NetAnalyzers' package provides. Update or remove this package reference. [#903](https://github.com/microsoft/binskim/pull/903)
```

Comment:
The language is inconsistent with other DEP updates. We should just say NetAnalyzers updated from 7.0.0 to 7.0.1

Reply:
Ok, changed to be consistent.

Reply:
@schlaman-ms Can you add "from 7.0.0" before "to 7.0.1". Most of the other DEPs follow this convention, and it makes sense because it makes the update self contained.

Reply:
Ok, done. Yes like I had it originally but with other wording.